### DISCUSSION TITLE
Menu on website broken

### DISCUSSION THREAD

**Discussion by eriklautenschlaeger** [python-pillow/Pillow#5904] [General]
_Can you confirm this? If so, then maybe it should move over to issues._

It seems as if the website for the reference is broken in at least one place. 
I attached two screenshots to show. 
I tried with a multitude of browsers, OS, and devices -- the result is the same everywhere. :-/

![2021-12-23 20_21_10-](https://user-images.githubusercontent.com/6643172/147284428-8be82c61-4392-4d6a-b1db-990f43a63f71.png)
![2021-12-23 20_21_39-Image Module — Pillow (PIL Fork) 8 4 0 documentation — Mozilla Firefox (Private ](https://user-images.githubusercontent.com/6643172/147284429-a72c212f-2f60-4fdd-8722-7305e12a9e43.png)
.

**Comment by radarhere** [comment]
Hi. Yes, I can replicate that problem. However, I can only replicate it for 'stable', which is the last release. If I switch to 'latest', which is built from the main branch, things look better.

<img width="1193" alt="Screen Shot 2021-12-24 at 8 50 12 am" src="https://user-images.githubusercontent.com/3112309/147294493-2428ccc7-8d0c-4361-95f5-f75ca7f29ea4.png">
<img width="1199" alt="Screen Shot 2021-12-24 at 8 51 51 am" src="https://user-images.githubusercontent.com/3112309/147294496-deedad46-93cb-4030-9238-6fd5b718eba1.png">

I presume the difference is due to #5821, and that 'stable' will look better after Pillow 9.0.0 is released, due out on January 2.

**Reply by radarhere** [reply]
Pillow 9.0.0 has been released, and I can no longer replicate this problem on 'stable'.
